llvm/lib/Target/LoongArch/LoongArchFloat32InstrInfo.td

```diff
@@ -126,6 +126,23 @@ def PseudoST_CFR : Pseudo<(outs),
 let hasSideEffects = 0, mayLoad = 1, mayStore = 0 in
 def PseudoLD_CFR : Pseudo<(outs CFR:$ccd),
                           (ins GPR:$rj, grlenimm:$imm)>;
+
+// SET_CFR_{FALSE,TRUE}
+// These instructions are defined in order to avoid expensive check error when
```

Comment:
What error is caused by not defining these shortcut nodes? Maybe you mean "... in order to avoid expensive checks if regular instruction patterns are used"?

Reply:
If use regular instruction will generate the following errors:
```
*** Bad machine code: Using an undefined physical register ***
- function:    test                                           
- basic block: %bb.0  (0x8deaa0)                              
- instruction: $fcc1 = FCMP_CAF_S $f0, $f0                    
- operand 2:   $f0                                            
```

Reply:
Hmm that seems something to fix... maybe later. (I don't have much free time recently for that, unfortunately.)